k8s/helm-charts/seldon-core-v2-setup/templates/seldon-v2-components.yaml

```diff
@@ -944,6 +944,15 @@ spec:
         image: '{{ .Values.envoy.image.registry }}/{{ .Values.envoy.image.repository
           }}:{{ .Values.envoy.image.tag }}'
         imagePullPolicy: '{{ .Values.envoy.image.pullPolicy }}'
+        lifecycle:
+          preStop:
+            exec:
+              command:
+              - /bin/sh
+              - -c
+              - |
+                echo -ne "POST /healthcheck/fail HTTP/1.1\r\nHost: localhost\r\nConnection: close\r\n\r\n" > /dev/tcp/localhost/9901
```

Comment:
what is ip 9901 (admin?) can this change?

Reply:
Yes, it's the admin interface. Made it configurable.